package/kernel/linux/modules/netdevices.mk

```diff
@@ -991,3 +991,19 @@ define KernelPackage/bnx2/description
 endef
 
 $(eval $(call KernelPackage,bnx2))
+
+
+define KernelPackage/bnx2x
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=QLogic 5771x/578xx 10/20-Gigabit ethernet adapter driver
+  DEPENDS:=@PCI_SUPPORT +bnx2x-firmware +kmod-lib-crc32c +kmod-mdio +kmod-ptp
```

Comment:
https://github.com/torvalds/linux/blob/master/drivers/net/ethernet/broadcom/Kconfig#L134

mentions zlib as well?

I've added the dependency and did some reshuffling. Please check.

https://git.openwrt.org/?p=openwrt/staging/chunkeey.git;a=commit;h=188ae12f39070d53c86d4f32fdb56dc40f9ea1c8

Reply:
will do, thanks

Reply:
@chunkeey, looks good to me :+1: